x-pack/plugins/apm/public/components/app/mobile/transaction_overview/index.tsx

```diff
@@ -63,6 +72,32 @@ export function MobileTransactionOverview() {
       <EuiFlexItem>
         <EuiHorizontalRule />
       </EuiFlexItem>
+      <EuiFlexItem>
+        <EuiFlexGroup>
+          <EuiFlexItem grow={8}>
+            <EuiPanel hasBorder={true}>
```

Comment:
What do you think about adding both `GeoMap` and `MobileLocationStats` in `EuiPanel`? 

![image](https://user-images.githubusercontent.com/3369346/217229407-572bd147-f80b-49a6-858b-35c1e3430be2.png)